yarn.lock

```diff
@@ -8447,14 +8456,14 @@ __metadata:
   linkType: hard
 
 "@prisma/generator-helper@npm:^4.0.0":
-  version: 4.12.0
-  resolution: "@prisma/generator-helper@npm:4.12.0"
+  version: 4.13.0
```

Comment:
I regenerated yarn.lock after pulling latest submodules and it seems to update these cc @emrysal @zomars